src/main/java/org/broadinstitute/hellbender/tools/picard/analysis/CollectOxoGMetrics.java

```diff
@@ -229,7 +228,7 @@ protected Object doWork() {
         }
         iterator.setSamFilters(filters);
 
-        log.info("Starting iteration.");
+        logger.info("Starting iteration.");
```

Comment:
What's the rational for using the base class logger here but declaring a new one in other places?  I think we should probably prefer declaring a new one in every specific class so the messages are more specific, but I suspect the existing code is just not consistent about this.